### PR TITLE
GC - use new group label components for overlap tab and survival tab

### PR DESCRIPTION
**Pull request by adamabeshouse**
Addresses https://github.com/cBioPortal/cbioportal/issues/6218

image

image

image

image

### CODE REVIEW

**Comment by jjgao** [comment]
@adamabeshouse looks good overall. For UpSet plot, I am wondering if it's possible (and looks good) to just add a colored square before the group name?

**Comment by adamabeshouse** [comment]
@jjgao how does this look?
image

**Comment by jjgao** [comment]
@adamabeshouse I like the square version. Maybe don't worry about the ticks in clinical tab and I think other tabs look good.

File: src/pages/groupComparison/Survival.tsx [hunk 1]
```diff
@@ -75,7 +79,7 @@ export default class Survival extends React.Component<ISurvivalProps, {}> {
                             name,
                             color: blendColors(partitionGroupUids.map(uid => uidToGroup[uid].color)),
                             value,
-                            legendText: name
+                            legendText: JSON.stringify(partitionGroupUids),
```
Comment (alisman) [inline src/pages/groupComparison/Survival.tsx]: just want to confirm this is intended

Comment (adamabeshouse) [reply src/pages/groupComparison/Survival.tsx]: yes its a bit weird but that's how I'm passing it through the system I have rigged up

**Pull request merged**